include/aws/s3/private/s3_client_impl.h

```diff
@@ -58,6 +58,32 @@ struct aws_s3_endpoint_options {
 
     /* HTTP port override. If zero, determine port based on TLS context */
     uint16_t port;
+
+    /**
+     * Optional.
+     * Proxy configuration for http connection.
```

Comment:
can we add a more explicit comment on relationship between proxy options and ev proxy options and what happens when users set both

Reply:
Thanks, fixed.